Cargo.toml

```diff
@@ -13,7 +13,7 @@ dirs        = "4.0"
 env_logger  = "0.9.0"
 lazy_static = "1.4.0"
 log         = "0.4.17"
-regex       = { version = "1.5.6", default-features = false }
+regex       = { version = "1.6.0", default-features = true }
```

Comment:
Which features do we need?

Reply:
the version does not really matter, but we need to  set `default-features` to true in order to use `unicode regex`, else, we will get this error

```
Regex failed
```

but i think the later the version, the better it gets so i pick the latest one

Reply:
ok